docs/core/deploying/runtime-patch-selection.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Runtime roll forward for .NET Core self-contained app deployments.
 description: Learn about dotnet publish changes for self-contained deployments.
-author: KathleenDollard
+author: kathleendollard
```

Comment:
Most of the noise seems to be from it lowercasing author names

Reply:
But her GitHub ID is KathleenDollard

Reply:
Might be a bug with the docs pack. I'm going to split this PR to just include the Redirect stuff, and I'll PR the links changes separately too. I'll open an issue on their repo

Reply:
@mairaw rebased out these changes for now and opened https://github.com/Microsoft/vscode-docs-authoring/issues/203